docs/content/getting-started/installing-viewer.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Installing the Rerun Viewer
+order: 0
+---
+
+The [Rerun Viewer](../reference/viewer/overview.md) can be installed independent of the SDK language you're using.
```

Comment:
I think we should make a note here that the viewer and SDK versions need to match.